mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -1785,6 +1785,43 @@ func TestAccComputeInstanceConfidentialInstanceConfigMain(t *testing.T) {
 	})
 }
 
+<% unless version = 'ga' -%>
```

Comment:
`<% unless version == 'ga' -%>`

This messed up generation and removed 1500+ lines in beta:
https://github.com/GoogleCloudPlatform/magic-modules/pull/9250#issuecomment-1761677021